src/components/common/sidepanegrid.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from 'react';
+import MenuCard from './menucard';
+import { IMenu } from '../../@types/menu';
+
+type Props = {
+    menuList: Array<IMenu>;
+    selectedMenu: number;
+    setSelectedMenu: (a: number) => void;
+};
+
+const SidePaneGrid: FC<Props> = ({ menuList, selectedMenu, setSelectedMenu }) => {
+
+    return (
+        <div className='transition-all ease-in-out duration-100 max-w-[350px]  border-slate-200 shadow-slate-200 border-solid border-r min-h-[calc(100vh-2.6rem)] z-10 bg-white' >
+            <div className='flex flex-wrap p-1'>
```

Comment:
merge divs in lines 14 and 15

Reply:
This is not yet addressed.

Reply:
Some styling issues are coming after merging those divs